webpack.dev.config.js

```diff
@@ -1,7 +1,15 @@
 const { merge } = require('webpack-merge');
 const commonConfig = require('./webpack.common.config');
 
-module.exports = merge(commonConfig, {
-  mode: 'development',
-  devtool: 'eval-source-map',
+const configuration = [];
```

Comment:
Which are the cases where we use `npm run dev` xD?

Reply:
`npm run build` generates the minify version of the files with the `npm run dev`  you can generate files that are easier to read and if I'm not wrong that generates the whole file tree in the browser console instead of just one js file, hot reloading but I think that's not working